### ISSUE TITLE
Add Basic Viewer for graphics 

### ISSUE THREAD

**Issue by nesterone** [zorko-io/zorko-platform#324]
**Problem:**

We don't have a UI for viewing graphics

**Solution:**

Add simple viewer for vega/vega-lite content

**Technical Solution:**

- add new feature 'viewer'
- redirect user from preview list to viewer page
- make sure that user can go back to preview list
- use 'previewUrl'  as an endpoint for getting particular content in viewer

**Comment by nesterone** [comment]
Simple sketch

[Zorko Viewer.pdf](https://github.com/zorko-io/zorko-platform/files/6328914/Zorko.Viewer.pdf)

**Comment by nesterone** [comment]
Endpoint for getting list of preview is `http://localhost/api/v1/previews`
each item has property a property `previewUrl`, for example 
```
{

...

  "previewUrl":"/api/v1/specs/2e3140e8-ed62-4843-929f-65becf347721",
...

}

```

it points to concrete specification content, so if you would do call 

GET `http://localhost/api/v1/specs/2e3140e8-ed62-4843-929f-65becf347721` 
it returns specification content

**Comment by nesterone** [comment]
bug to fix later on #325
